llvmcpy/llvm.py

```diff
@@ -161,7 +161,7 @@ def header():
         zeroth_argument = ["self"] if is_class_method else []
         # Discard None arguments, they have been removed
         args = filter(lambda x: x is not None, function_arguments)
-        function_arguments_str = ", ".join(zeroth_argument + args)
+        function_arguments_str = ", ".join(zeroth_argument + list(args))
```

Comment:
It was required to get any py3k tests to pass, that's why it was included here.

I can put it in a separate PR, I just figured it would make sense to have all the tests green.  If I remove it, should I disable the py3k tests?